docs/docs-content/clusters/cluster-management/cluster-updates.md

```diff
@@ -140,3 +142,123 @@ You can follow these steps to validate all cluster update approaches.
 3. Select the cluster you updated, and navigate to the **Profile** tab of the cluster.
 
 4. Confirm that the cluster is now using an updated profile.
+
+## Palette Reconciliation Behavior
+
+When cluster profile updates are applied, Palette will apply the changes to the cluster but will not delete any
+resources that are no longer in use. Depending on the situation, you can choose to keep or remove these resources. To
+remove the resources, utilize one of the following two methods.
+
+The first method would be a clean deployment with a new cluster profile. Once the new cluster and cluster profile are
+deployed, the previous version could be deleted. The challenge, however, is when particular resources are in use.
```

Comment:
```suggestion
deployed, the previous version could be deleted. However, this may have unintended downstream effects because it does not preserve any resource that is currently in use.
```
The last sentence seems a bit vague to me. I rewrote it to be a little more explicit what the risks are (if these are what you meant when you say 'when particular resource is in-use'.